DQM/L1TMonitor/src/L1TObjectsTiming.cc

```diff
@@ -92,6 +95,16 @@ void L1TObjectsTiming::bookHistograms(DQMStore::IBooker& ibooker, const edm::Run
 
   // Subsystem Monitoring and Muon Output
   std::string bx_obj[5]={"minus2","minus1","0","plus1","plus2"};
+  
+  ibooker.setCurrentFolder(monitorDir_);
```

Comment:
Place the denominator histograms in the subdirectory of the corresponding L1T object. Do not forget to also update the paths for the DQMGenericClients in L1TObjectsTimingClient_cff.py